gnocchi/tests/test_carbonara.py

```diff
@@ -105,21 +105,6 @@ def test_duplicate_timestamps(self):
 
 
 class TestAggregatedTimeSerie(base.BaseTestCase):
-    @staticmethod
-    def test_base():
```

Comment:
you deleting this because it tests nothing?

Reply:
Yes. It existed because the `3` or `"4s"` once were converted at `__init__` time to a `numpy.timedelta64` (or even a pandas type) but that's not the case anymore as this is done way before in the current code (which is faster). So rather than fixing it stupidly I went ahead and just dropped it.